packages/neon_framework/lib/src/blocs/apps.dart

```diff
@@ -268,7 +266,9 @@ class _AppsBloc extends InteractiveBloc implements AppsBloc {
       subject: apps,
       getRequest: account.client.core.navigation.$getAppsNavigation_Request,
       serializer: account.client.core.navigation.$getAppsNavigation_Serializer(),
-      unwrap: (response) => response.body.ocs.data,
+      unwrap: (response) => response.body.ocs.data.rebuild(
+        (b) => b..sort((a, b) => (a.getOrder() ?? 100).compareTo(b.getOrder() ?? 100)),
```

Comment:
1. why is `100` the number used?
2. please extract this into a separate constant
3. let `getOrder()` return the default order.

Reply:
The order is actually not nullable, because it is a oneOf (but I just randomly chose 100 as there is no value that naturally makes sense if the order were missing).